### ISSUE TITLE
Can you help to check what is wrong with my training?

### ISSUE THREAD

**Issue by MaitaYuki** [TencentAILabHealthcare/ConCL#4]
Hi @Jiawei-Yang,

Thanks for the great work and sharing your code!

I tried to reproduce your training results. I downloaded your training dataset and followed your instructions to train on 2 GPUs using the command "bash tools/dist_train.sh configs/NCT_pretraining/concl/r18_bs256_ep200.py 2" (I set batch_size to 64 and imgs_per_gpu= batch_size//2).
The strange thing is that the training loss has not been reduced, and it only takes 10 hours to train 200 epochs.
The initial loss in epoch 1:  loss: 7.0236, instance_l: 7.0236, concept_l: 0.0000
The loss in the end of epoch  200: loss: 8.2989, instance_l: 9.6324, concept_l: 8.2989
Attached is my training log.

Could you guess what went wrong?
[train.log](https://github.com/TencentAILabHealthcare/ConCL/files/11356999/train.log)
